internal/model/rwfolder.go

```diff
@@ -54,6 +54,19 @@ var (
 	errNoDevice = errors.New("no available source device")
 )
 
+const (
+	dbUpdateHandleDir = iota
+	dbUpdateDeleteDir
+	dbUpdateHandleFile
+	dbUpdateDeleteFile
+	dbUpdateShortcutFile
+)
+
+type dbUpdateJob struct {
+	file    protocol.FileInfo
+	jobType int
```

Comment:
It doesn't seem like we actually use this on the receiving side, as in the end it mostly checks for deleted/directory/invalid based on the FileInfo instead?